tests/cxx/final_forward_declare.h

```diff
@@ -0,0 +1,30 @@
+//===--- final_forward_declare.h - test input file for iwyu ---------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
```

Comment:
Please use include guards instead of `#pragma once`, see other tests for naming.